RELEASING.md

```diff
@@ -1,22 +1,24 @@
-# Releasing a new version
-## Optional - Updating dependencies
-Makefile: 
-- BUILD_CONTAINER_NAME?=calico/build:v0.18.0 - Currently, the startup.py script relies on the Python version of libcalico
-- FELIX_CONTAINER_NAME?=calico/felix:2.0.0-beta.3
-- LIBNETWORK_PLUGIN_CONTAINER_NAME?=calico/libnetwork-plugin:v1.0.0-beta-rc2
-- Also, less commonly Bird, GoBGP and confd
+# Release process
+
+## Resulting artifacts
+Creating a new release creates the following artifacts
+* `calico/node:$VERSION` container image
+* `calico/ctl` container image
+* `calicoctl`  binary (stored in the `dist` directory.
 
-glide.yaml
-- Update any Go dependencies for Calicoctl here, e.g. libcalico-go
-    - After updating glide.yaml run `glide up -v`
+## Preparing for a release
+Ensure that the branch you want to release from (typically master) is in a good state.
+e.g. Update the libcalico-go pin to the latest release in glide.yaml and run `glide up -v`, create PR, ensure test pass and merge.
+or update other dependencies in the `Makefile` 
+- `BUILD_CONTAINER_NAME?=calico/build:v0.18.0` - Currently, the startup.py script relies on the Python version of libcalico
+- `FELIX_CONTAINER_NAME?=calico/felix:2.0.0`
+- `LIBNETWORK_PLUGIN_CONTAINER_NAME?=calico/libnetwork-plugin:v1.0.0`
+- Also, less commonly Bird, GoBGP and confd
 
-After performing these updates, make sure they are tested and commit the changes to `master`
+You should have no local changes and tests should be passing.
 
-## Releasing calico-containers (`calico/node` and `calicoctl`
+## Creating the release
 1. Choose a version e.g. `export VERSION=v1.0.0`
```

Comment:
Do we need to have an environment variable set *and* pass it in on the make target?  I'd rather we just forced the user to specify it on the make target and got rid of the environment variable if possible.